tests/integration/helm/helm_upgrade_test.go

```diff
@@ -0,0 +1,170 @@
+// +build integ
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework/image"
+
+	"istio.io/istio/pkg/test/env"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/helm"
+)
+
+var (
+	// previousChartPath is path of Helm charts for previous Istio deployments.
+	previousChartPath        = filepath.Join(env.IstioSrc, "tests/integration/helm/testdata/")
+	previousSupportedVersion = "1.8.1"
+
+	istioDockerHub = "docker.io/istio"
```

Comment:
should make this `gcr.io/istio-release` to avoid rate-limiting

Reply:
Done.